testsuite/MDAnalysisTests/analysis/test_distances.py

```diff
@@ -21,91 +21,71 @@
 #
 from __future__ import print_function, absolute_import
 
+import pytest
 import scipy
 import scipy.spatial
 
 import MDAnalysis
-from MDAnalysisTests import module_not_found
 from MDAnalysisTests.datafiles import GRO
-from MDAnalysisTests.util import block_import
 
 import MDAnalysis.analysis.distances
 
-from numpy.testing import TestCase, assert_equal, dec
+from numpy.testing import assert_equal
 import numpy as np
 
-import warnings
-import sys
-
-
-class TestContactMatrix(TestCase):
-
-    def setUp(self):
-        import MDAnalysis.analysis.distances
-        self.coord = np.array([[1, 1, 1],
-                                  [5, 5, 5],
-                                  [1.1, 1.1, 1.1],
-                                  [11, 11, 11],  # neighboring image with pbc
-                                  [21, 21, 21]],  # non neighboring image with pbc
-                                 dtype=np.float32)
-        self.box = np.array([10, 10, 10], dtype=np.float32)
-        self.shape = (5, 5)
-        self.res_no_pbc = np.array([[1, 0, 1, 0, 0],
-                                       [0, 1, 0, 0, 0],
-                                       [1, 0, 1, 0, 0],
-                                       [0, 0, 0, 1, 0],
-                                       [0, 0, 0, 0, 1]], dtype=np.bool)
-        self.res_pbc = np.array([[1, 0, 1, 1, 1],
-                                    [0, 1, 0, 0, 0],
-                                    [1, 0, 1, 1, 1],
-                                    [1, 0, 1, 1, 1],
-                                    [1, 0, 1, 1, 1]], dtype=np.bool)
+
+
+class TestContactMatrix(object):
+    coord = np.array([[1, 1, 1],
+                      [5, 5, 5],
+                      [1.1, 1.1, 1.1],
+                      [11, 11, 11],  # neighboring image with pbc
+                      [21, 21, 21]],  # non neighboring image with pbc
+                     dtype=np.float32)
+
+    box = np.array([10, 10, 10], dtype=np.float32)
+    shape = (5, 5)
+    res_no_pbc = np.array([[1, 0, 1, 0, 0],
+                           [0, 1, 0, 0, 0],
+                           [1, 0, 1, 0, 0],
+                           [0, 0, 0, 1, 0],
+                           [0, 0, 0, 0, 1]], dtype=np.bool)
+
+    res_pbc = np.array([[1, 0, 1, 1, 1],
+                        [0, 1, 0, 0, 0],
+                        [1, 0, 1, 1, 1],
+                        [1, 0, 1, 1, 1],
+                        [1, 0, 1, 1, 1]], dtype=np.bool)
+
 
     def test_np(self):
-        contacts = MDAnalysis.analysis.distances.contact_matrix(
-            self.coord, cutoff=1, returntype="numpy")
-        assert_equal(contacts.shape, self.shape,
-                     "wrong shape (should be {0})".format(self.shape))
+        contacts = MDAnalysis.analysis.distances.contact_matrix(self.coord, cutoff=1, returntype="numpy")
+        assert contacts.shape == self.shape, "wrong shape (should be {0})".format(self.shape)
```

Comment:
This looks like you break the 80 char line limit. Also these tests look like they could be parametrized.